Description: add encoding directive to POD
 Otherwise we get a POD ERROR:
    Around line 1209:
        Non-ASCII character seen before =encoding in 'Trmaƒç'. Assuming UTF-8
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2015-11-08

--- a/lib/IPTables/Parse.pm
+++ b/lib/IPTables/Parse.pm
@@ -995,6 +995,8 @@
 1;
 __END__
 
+=encoding UTF-8
+
 =head1 NAME
 
 IPTables::Parse - Perl extension for parsing iptables and ip6tables policies
